apps/emails-and-messages/src/components/temp.tsx

```diff
@@ -0,0 +1,160 @@
+const x = () => (
+    <SectionWithDescription
+        title="Events"
+        description={
+          <>
+            <Text display="block">
```

Comment:
you can also write semantic `as="p"` that will render it properly, because `<p>` is block too